src/ProfilesPage/ProfilePage.js

```diff
@@ -22,14 +22,26 @@ import UserInfo from "./components/UserInfo";
 import ProfileConnections from "./components/ProfileConnections";
 import ProfileFeed from "./components/ProfileFeed";
 import EditPage from "../EditPage/EditPage";
+import Modal from 'react-responsive-modal';
+
 
 const userId = sessionStorage.getItem("userId");
 const adminUser = sessionStorage.getItem("adminUser");
 const admin = sessionStorage.getItem("isAdmin");
 // var editButtonClicked = false;
 
+const modStyle = {
```

Comment:
This doesn't appear to be used anywhere. Can we get rid of it?

Reply:
I got rid of it. I think I was trying to add some style to the modal.